google-datacatalog-connectors-commons/src/google/datacatalog_connectors/commons/datacatalog_facade.py

```diff
@@ -123,23 +138,95 @@ def __entry_was_updated(cls, current_entry, new_entry):
             new_update_time != 0 and current_update_time != new_update_time
 
         return updated_time_changed or not cls.__entries_are_equal(
-            current_entry, new_entry)
+            current_entry=current_entry, new_entry=new_entry)
 
     @classmethod
-    def __entries_are_equal(cls, entry_1, entry_2):
+    def __entries_are_equal(cls, *, current_entry, new_entry):
+        """
+        Compares if two entries are equal.
+        Since the args order must be respected, this method uses
```

Comment:
Same suggestion from above...

Reply:
Removed the key-args, this is not longer needed.